.gitattributes

```diff
@@ -22,3 +22,4 @@ UPGRADE.md export-ignore
 /public export-ignore
 /tests export-ignore
 /templates export-ignore
+/assets export-ignore
```

Comment:
See https://github.com/sulu/sulu-minimal/pull/174/files#r284544230
I think we should not ignore in this folder, so that it can be reused in sulu-minimal.

Reply:
We have decided to duplicate the code for now, because it allows more fine-grained control on what should be included in the build (e.g. if somebody only supports modern browser, they can omit the polyfills this way).